### PR TITLE
Fix/Slop-57/expand-size-of-blog-post-body

### PR DESCRIPTION
**Pull request by ATribbey**
## Describe your changes
Adjusted the post model in order to allow for larger posts. 

Brennan (bduckdev) was instrumental in resolving this issue, as he assisted me in combing through keystone docs and finding what changes need to be made. 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If there are changes to custom resolvers, I have added / updated automated tests
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by bduckdev** [comment]
Functions as expected. 

Only thing worth noting is that as we discussed, adding "isNullable" to the content value makes it so that in the backend UI, an additional checkbox no appears in the content portion of the post which asks if the value of content is null.

**Review by fdlai — COMMENTED** [review]
Great job! It works as I show in the screenshot. My only reservation, is that this branch forces us to reset the database, and I'm not sure if we want to do that or not? If I hear back from @tomershvueli that it's fine, then I'll go ahead and approve! Oh, and there may be some style changes that need to be done in how a long blog is displayed, but that can be a new task.

**Review by fdlai — COMMENTED** [review]
Hey! I pulled your changes. But it's still telling me I need to reset the database. Does it not do that for you?

**Comment by fdlai** [comment]
@ATribbey Not sure if you saw my last review from 2 weeks ago, but I just tested this again, and it's still prompting me to reset the database.

**Comment by ATribbey** [comment]
> @ATribbey Not sure if you saw my last review from 2 weeks ago, but I just tested this again, and it's still prompting me to reset the database.

Hey @fdlai I just merged with main and had no issue applying migrations without DB reset. Would you mind giving it another shot for me?

image

File: migrations/20240510231712_/migration.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE `post` MODIFY `content` TEXT NULL;
```
Comment (tomershvueli) [inline migrations/20240510231712_/migration.sql]: We've had issues with this in previous PR's, but `post` should be capitalized here to ensure we're altering the correct table.

**Review by bduckdev — APPROVED** [review]
Change works as described still. I see that you also did the changes requested by Tomers. Nice work!

**Review by marzahlde09 — APPROVED** [review]
Works as described!

**Review by tomershvueli — APPROVED** [review]
I synced this up with `main` since it was a few commits behind from some PR's I just merged in. This change brings about some minor UI issues, but that's a problem for another day. This is working well for me!

**Pull request merged**